dev/com.ibm.ws.concurrent.persistent_fat_multiple/fat/src/com/ibm/ws/concurrent/persistent/fat/multiple/DatabaseContainerUtil.java

```diff
@@ -0,0 +1,149 @@
+package com.ibm.ws.concurrent.persistent.fat.multiple;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.Statement;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+import com.ibm.websphere.simplicity.config.DataSource;
+import com.ibm.websphere.simplicity.config.DataSourceProperties;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.websphere.simplicity.config.dsprops.Properties;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_db2_jcc;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_microsoft_sqlserver;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_oracle;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.topology.impl.LibertyServer;
+
+public class DatabaseContainerUtil {
+	//Logging Constants
+	private static final Class<DatabaseContainerUtil> c = DatabaseContainerUtil.class;
+	private static final String nl = System.getProperty("line.separator");
+	
+	/**
+	 * Initializes SQL Server database.  Creates a database with a given name, and
+	 * runs the sp_sqljdbc_xa_install stored procedure to enable XA transactions.
+	 * 
+	 * @param cont - SQLServer Test Container
+	 * @param databaseName - Name to be given to database
+	 */
+	public static void initSQLServer(JdbcDatabaseContainer<?> cont, String databaseName) {
+		// Create Database
+        try (Connection conn = cont.createConnection("")) {
+            Statement stmt = conn.createStatement();
+            stmt.execute("CREATE DATABASE [" + databaseName + "];");
+            stmt.close();
+        } catch (Exception e) {
+        	Log.error(c, "initContainer", e);
```

Comment:
instead of burying this exception and the next one, we should let the errors get thrown out of the method so we don't attempt to run the tests later on and have them fail in non-obvious ways